meson_options.txt

```diff
@@ -16,6 +16,12 @@ option(
   description : 'generate man pages',
   value : 'auto',
 )
+option(
+  'tests',
+  type : 'boolean',
+  description : 'build tests',
+  value : 'true',
+)
```

Comment:
Options in alphabetical order, please? (That would currently put this one just after `selinux`, I think.)

Reply:
Done